Formula/sdns.rb

```diff
@@ -0,0 +1,50 @@
+class Sdns < Formula
+  desc "Privacy important, fast, recursive dns resolver server with dnssec support"
+  homepage "https://sdns.dev"
+  url "https://github.com/semihalev/sdns/archive/v1.1.5.tar.gz"
+  sha256 "1b874b5fe86e16a11f476f0c1626a9d5bca34e5d03759b6208a5311ae785edd3"
+  license "MIT"
+  head "https://github.com/semihalev/sdns.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "sdns"
+  end
+
+  plist_options startup: true
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{opt_bin}/sdns</string>
+            <string>-config</string>
+            <string>#{etc}/sdns.conf</string>
+          </array>
+          <key>RunAtLoad</key>
+          <true/>
+          <key>KeepAlive</key>
+          <true/>
+          <key>StandardErrorPath</key>
+          <string>#{var}/log/sdns.log</string>
+          <key>StandardOutPath</key>
+          <string>#{var}/log/sdns.log</string>
+          <key>WorkingDirectory</key>
+          <string>#{opt_prefix}</string>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    system "#{bin}/sdns", "-v"
```

Comment:
Version tests are bad tests: https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Based on the `sdns` docs, I suggest using `-config` to create a config file, then at least test that it exists.

Reply:
I updated based on your suggestion.